nidmviewer/convert.py

```diff
@@ -24,7 +25,10 @@ def parse_coordinates(coordinates):
     count=0
     coordinate_df = pandas.DataFrame(columns=["x","y","z"])
     for coordinate in coordinates:
-        coordinate_df.loc[count] = [x.strip() for x in coordinate.strip("]").strip("[").split(",")]
+        if not isinstance(coordinate, float64) or not isnan(coordinate):
```

Comment:
If the coordinates exist then yes they will always be string. But, if there are no peaks/clusters, the coordinates are set to "Nan", hence the check. See for example with http://neurovault.org/collections/1404/fsl_ds005_sub-01.nidm. Does that make sense?